Source/Laser/LaserParticleContainer.H

```diff
@@ -68,17 +60,17 @@ protected:
     std::string laser_name;
 
 private:
-
     // runtime paramters
-    laser_t                   profile     = laser_t::Null;
     amrex::Vector<amrex::Real> position;
     amrex::Vector<amrex::Real> nvec;
     amrex::Vector<amrex::Real> p_X;
     amrex::Vector<amrex::Real> stc_direction;
+    amrex::Real theta_stc;
```

Comment:
Maybe we can add a `//! comment to each variable`?

Reply:
I've added a comment to all these variables. `theta_stc` is actually computed using runtime parameters, so I've moved it below.